data/Yang_2023/metadata.yml

```diff
@@ -84,7 +84,11 @@ dataset:
   custom_R_code: '
     data %>%
       mutate(
-        seedbank_longevity = seed_longevity_categorical
+        seedbank_longevity = seed_longevity_categorical,
+        reproductive_maturity_secondary_entity_type = ifelse(is.na(reproductive_maturity_secondary_entity_type), 
```

Comment:
I can't find any rows where `reproductive_maturity_secondary_entity_type` is NA while `reproductive_maturity_entity_type` isn't empty, nor for `reproductive_maturity_secondary_value_type`.